modules/040-node-manager/webhooks/validating/node_group

```diff
@@ -86,13 +86,6 @@ EOF
     return 0
   fi
 
-  if [[ "$minPerZone" -eq 0 && "$maxPerZone" -gt 0 ]]; then
```

Comment:
Is it possible to improve validating webhook to forbid node group creation/updating if there is an instance class without known flavor or specified capacity and the minimum number of instances per zone equals 0 instead of deleting the validation completely?

Reply:
Don't think so. Cause then this hook have to know everything not about node-group but about all instance classes. Have to known about their instance types and so on.  It won't be the validating webhook, it couldnt contain such a colpicated logic, but some small controller. We are making all this checks in the hook and fire an error if something goes wrong

Reply:
Ok, it also makes sense. I was just curious maybe you have some thought how to do it)